distribution/docker/Dockerfile

```diff
@@ -58,9 +79,9 @@ RUN if [ "$TARGETARCH" = "arm64" ]; then \
 RUN addgroup -S -g 1000 druid \
  && adduser -S -u 1000 -D -H -h /opt/druid -s /bin/sh -g '' -G druid druid
 
-COPY --chown=druid:druid --from=extractor /opt /opt
-COPY ./docker/druid.sh /druid.sh
-COPY ./docker/peon.sh /peon.sh
+COPY --chown=druid:druid --from=builder /opt /opt
+COPY distribution/docker/druid.sh /druid.sh
```

Comment:
same here too.

Reply:
This path matches the one used upstream. 

The reason we used a different one in the past is because the maven plugin ran the docker build relative to the distribution directory instead of the repository root. 

Now that we no longer use the maven plugin we can use the same path as upstream which assumes you run docker from the repo root.